nullaway/src/test/java/com/uber/nullaway/NullAwayFrameworkTests.java

```diff
@@ -365,4 +365,75 @@ public void systemConsoleNullable() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void mapGetOrDefault() {
```

Comment:
```suggestion
  public void mapGetOrDefaultNoRestrictiveAnnotations() {
```

(maybe? Feels less easy to miss-parse mentally than `WithoutAcknowledgeRestrictiveAnnotations`)